src/base/geom/affinetransform.h

```diff
@@ -15,33 +15,25 @@ class Polygon;
 class AffineTransform
 {
 public:
-	Geom::Point offset;
-	double scale;
-	double rotate;
+	double m[2][3];
```

Comment:
It can be made as private member. Then it need `JScriptOutputCanvas::transform` to be a friend function.

Reply:
JScriptOutputCanvas is in the application logic, wouldn't reference here in this very low level generic class.

Reply:
OK :)

Reply:
I can make it private with a const ref  getter:
```cpp
const std::array<std::array, 3>, 2> getMatrix() const;
```